src/elements/checkbox/checkbox-panel/checkbox-panel.ts

```diff
@@ -51,6 +56,16 @@ export class SbbCheckboxPanelElement extends SbbPanelMixin(
     panelConnected: 'panelConnected',
   } as const;
 
+  /** Label size variant, either m or s. */
+  @property({ reflect: true })
+  public set size(value: SbbCheckboxPanelSize) {
+    this._size = value;
+  }
+  public get size(): SbbCheckboxPanelSize {
+    return this._size;
```

Comment:
I think you missed the inheritance from the group
If the group has size 'xs' I would fallback to 's' on the panel